src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -62,7 +62,7 @@ public function __construct(EventDispatcherInterface $dispatcher, ControllerReso
      */
     public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQUEST, $catch = true)
     {
-        $request->headers->set('X-Php-Ob-Level', ob_get_level());
+        $request->headers->set('X-Php-Ob-Level', (string) (ob_get_level()));
```

Comment:
you can also remove the extra brackets now

Reply:
OK, good idea!